packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -1156,7 +1156,7 @@ The first three parameters in the table below correspond to different ways to li
 | limit       | Specifies the number of results returned (maximum 200).                           | Query        | Number     | FALSE    |
 | after       | Specifies the pagination cursor for the next page of users.                                                                                     | Query        | String     | FALSE    |
 | sortBy      | Specifies field to sort by (for search queries only).                                                                                           | Search query | String     | FALSE    |
-| sortOrder   | Specifies sort order asc or desc (for search queries only).                                                                                     | Search query | String     | FALSE    |
+| sortOrder   | Specifies sort order asc or desc (for search queries only). Sorting is done in ASCII sort order (that is, by ASCII character value). Sorting is case insensitive.                                                                                                                                                    | Search query | String     | FALSE    |
```

Comment:
Maybe it's just me, the sentence seem a little repetitive - maybe "Sorting is done in ASII ..., and is case insensitive" or similar?
No more comment on the correctness though, I'll leave the detail up to you.